svthalia/concrexit
# Fix album processing

Closes #3410.



### Summary
The process of uploading an album is put into the background. Now, when a lot of photos are uploaded, the website doesn't give an error while the album still gets uploaded, which is confusing. 

### How to test

1. Go to admin control
2. Click on 'add album'
3. Add a big album and check if the website gives an error
